internal/pkg/agent/cmd/run.go

```diff
@@ -162,6 +167,12 @@ func runElasticAgent(ctx context.Context, cancel context.CancelFunc, override cf
 
 	l.Infow("Elastic Agent started", "process.pid", os.Getpid(), "agent.version", version.GetAgentPackageVersion())
 
+	// try early to check if running as root
+	isRoot, err := utils.HasRoot()
+	if err != nil {
+		return fmt.Errorf("failed to check for root permissions: %w", err)
```

Comment:
The others you added used the word `administrator` and this one used `root`. I believe that was just because this was moved. I would like to see this consitent, maybe `root/Administrator` would be better and work for unix and windows.

Reply:
cleaned up a bit in b50857049b98e7eb7f2d04fa6e3eeeea3f14ce7b